conanfile.py

```diff
@@ -17,14 +17,15 @@ class KeychainConan(ConanFile):
     _source_subfolder = "source_subfolder"
     _build_subfolder = "build_subfolder"
 
-    def build_requirements(self):
-        if self.settings.os == "Linux":
-            self.build_requires("pkg-config_installer/0.29.2@bincrafters/stable")
-
     def requirements(self):
-        if self.settings.os == "Linux":
-            self.requires("glib/2.58.3@bincrafters/stable")
-            # TODO: keychain also requires 'libsecret-1' which is not in conan
+        pass
+        # Note: keychain requires 'libsecret-1' on Linux which is not in Conan. libsecret must
+        # therefore be installed on the system manually.
+        # It further requires `glib-2.0` (via libsecret), which is in Conan. However, we cannot use
+        # it as it might be incompatible with the system-installed version of libsecret.
+        # For both of these dependencies, keychain relies on pkgconfig in order to find them. Once
+        # both of the dependencies are available in Conan, we can use `pkg-config_installer` to find
+        # them via pkgconfig.
```

Comment:
Just an idea: Maybe we could validate the existence of both `libsecret-1` and `glib-2.0` via a call to `pkg-config` here? Something along those lines:
```
pkg-config --validate glib-2.0 --silence-errors
pkg-config --validate libsecret-1 --silence-errors
```

Reply:
`tools.PkgConfig("libsecret-1")` should do the trick